index.html

```diff
@@ -161,7 +161,7 @@
           <div>
             Never use a reference manager again.
             Provide a DOI, ISBN, PubMed ID, URL, etc., and let Manubot generate and number citations for you.
-            Choose from <a href="https://www.zotero.org/styles">thousands of existing citation styles</a>.
+            Choose from <a href="https://citationstyles.org/">thousands of existing citation styles</a>.
```

Comment:
I wonder if we could keep the link on `thousands` to https://www.zotero.org/styles and then link `citation styles` to https://citationstyles.org/? The zotero page is nice by showing a searchable list of all the supported style and how they render things. However, I agree it would also be good to link to the CSL homepage.

Reply:
If you could use

> Choose from <a href="https://www.zotero.org/styles">thousands</a> of existing <a href="https://citationstyles.org/">Citation Style Language</a> citation styles</a>

that would be perfect.

(cc @adam3smith)

Reply:
We tend to try to keep the description text on the manubot.org homepage rather accessible and high-level, so you'll notice in many places we're a bit vague (with hyperlinks providing additional details). For example, we write things like "All in a [central location](https://github.com/)." So I'm just a bit worried that writing "Citation Style Language citation styles" is a bit verbose and out of character of the rest of the homepage. 

What about adding `title="Citation Style Language"` to the https://citationstyles.org/ link on "citation styles". This way users could see the full name on hover?

Reply:
Sure. Updated the PR to your suggestion.